udfs/community/test_cases.js

```diff
@@ -2308,3 +2308,43 @@ generate_udf_test("interval_micros", [
     expected_output: `CAST(-86400000000 AS INT64)`
   },
 ]);
+
+generate_udf_test("uint256_add", [
```

Comment:
Please add some more test cases to coverage like the following:
- null inputs
- divide by zero
- empty string input